test/click_cli/test_cshow_config.py

```diff
@@ -0,0 +1,48 @@
+from textwrap import dedent
+
+import pytest
+from mock import patch
+
+
+@pytest.fixture
+def config_file(tmpdir):
```

Comment:
Doesn't look like this is used.

Reply:
Correct. Again comes from the original test suite.

I will either remove the extraneous fixture or use it to shorten the test case itself.I guess you prefer removing the fixture.